### PR TITLE
Create Model Product,Category,Order 

### PR DESCRIPTION
**Pull request by thainv-0029**
Create model product ,category,order to get data from DB.

### CODE REVIEW

**Comment by thainv-0029** [comment]
ready

**Comment by sonnnd-1393** [comment]
dùng bower hoặc yarn đi em

https://viblo.asia/p/quan-ly-css-va-js-danh-cho-nguoi-moi-bat-dau-gGJ59b8DKX2

**Comment by thainv-0029** [comment]
ready

File: .bowerrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "directory": "public/bower_components/"
+}
```
Comment (sonnnd-1393) [inline .bowerrc]: dòng trắng cuối file

**Comment by thainv-0029** [comment]
ready

**Pull request merged**